docs/sources/write/links/_index.md

```diff
@@ -100,15 +100,22 @@ If there is no _`<KEY>`_ in the `refs` field, or there is no `refs` field in the
 
 Use a URL.
 
-If you are linking to versioned documentation, use a URL with version substitution syntax instead of the version path element.
+If you're linking to versioned documentation within your own documentation, use a URL with version substitution syntax instead of the version path element.
 For example, in Grafana, use `<GRAFANA_VERSION>` instead of `latest` in the URL `https://grafana.com/docs/grafana/latest/`.
 
+If you're linking to versioned documentation in another project, use version substitution if you have version compatibility constraints between the projects.
+If you don't have those constraints, use `latest` in the URL.
```

Comment:
It's probably worth mentioning here that if you're linking to another project, the version should be _for the project you are linking to_.  I had to fix a couple of broken links to Loki content that used <GRAFANA_VERSION> instead of <LOKI_VERSION> as the substitution.

Reply:
Applied in e7025922a1e2b38469689c3ba465579a019389a4